api/swagger.yml

```
@@ -199,6 +199,40 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ReportPreviewResponse"
+        400:
+          $ref: "#/components/responses/BadRequest"
+        403:
+          $ref: "#/components/responses/Forbidden"
+        500:
+          $ref: "#/components/responses/Internal"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)